doc/manual/src/json/store-object-info.md

```diff
@@ -0,0 +1,90 @@
+Info about a [store object].
+
+{{#include ./experimental-notice.md}}
+
+* `path`:
+
+  [Store path][store path] to the given store object.
+
+* `narHash`:
+
+  Hash of the [file system object] part of the store object when serialized as a [Nix Archive](#gloss-nar).
+
+* `narSize`:
+
+  Size of the [file system object] part of the store object when serialized as a [Nix Archive](#gloss-nar).
+
+* `references`:
+
+  An array of [store paths][store path], possibly including this one.
+
+* `ca` (optional):
+
+  Content address of this store object's file system object, used to compute its store path.
+
+[store path]: @docroot@/glossary.md#gloss-store-path
+[file system object]: @docroot@/store/file-system-object.md
+
+## Impure fields
+
+These are not intrinsic properties of the store object.
```

Comment:
What does "intrinsic" mean here? I'd like a bit of expansion.

Reply:
Added a second sentence explaining via an equivalent statement:

> In other words, the same store object residing in different store could have different values for these properties.